Practice/Shiny_App.R

```diff
@@ -0,0 +1,80 @@
+# Load packages-----------------------------------------------------------------
+library(tidyverse)
+library(shiny)
+library(shinythemes)
+library(shinyWidgets)
+
+
+# Build UI----------------------------------------------------------------------
+ui <- fluidPage(
+  
+  titlePanel("Mental Illness & Socioeconomic Status"),
+  
+  fluidRow(
+    column(5, offset = 1,
+           sliderTextInput("income", "Household Income",
+                           choices = c("$0-$9,999", "$10,000-$24,999", "$25,000-$49,999", "$50,000-$74,999", "$75,000-$99,999",
+                                       "$100,000-$124,999", "$125,000-$149,999", "$150,000-$174,999", "$175,000-$199,999", "$200,000+"),
```

Comment:
In response to your question:
> *How can I take the input from line 15-18 in the server to filter 'survey_data' based on the Household Income selected?*
```suggestion
                           choices = survey_data$`Household Income`,
```

Reply:
Not sure if this is what you're looking for?